templates/zerver/help/enable-night-mode.md

```diff
@@ -0,0 +1,8 @@
+# Enable night mode
+
+You may choose to enable night mode to make the background darker and the text lighter. Use the following procedure to enable night mode:
```

Comment:
The purpose of night mode is to "reduce the brightness of your screen lower than you can achieve with the default settings." Do you agree with that? I took it from [here](https://play.google.com/store/apps/details?id=pt.bbarao.nightmode&hl=en), so if you want to use that you'd probably have to reword it.

Also, I think this line should be line-wrapped (linter is going to give an error.)